packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -734,6 +734,7 @@ class DescopeWc extends BaseDescopeWc {
         window.location.origin,
       );
       window.close();
+      return;
```

Comment:
I thought that since we’re closing the window, there would be no need to return here
but I guess it’s needed after all

Reply:
Yes there seems to be a race there